ocaml/idl/ocaml_backend/gen_db_actions.ml

```diff
@@ -93,6 +93,8 @@ let dm_to_string tys : O.Module.t =
           "fun x -> x |> SecretString.rpc_of_t |> Rpc.string_of_rpc"
       | DT.Record _ ->
           failwith "record types never stored in the database"
+      | DT.Option (DT.Ref _ as ty) ->
+          "fun s -> set " ^ OU.alias_of_ty ty ^ "(Option.to_list s)"
```

Comment:
Might want to use `String.concat` because it is cheaper. Likewise below.

Reply:
Doesn't particularly matter as this runs at build time, but changed as suggested